lib/binary-buffer.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+module.exports = () => {
+    /**
+     * Converts a string into a buffer. Does not change the input if it is
+     * already a buffer.
+     *
+     * @param {(Buffer|string)} stringOrBuffer
+     * @return {(Buffer|string)}
+     */
+    function toBuffer(stringOrBuffer) {
+        if (!Buffer.isBuffer(stringOrBuffer)) {
+            stringOrBuffer = new Buffer(stringOrBuffer.toString(), "binary");
```

Comment:
It would be better to figure out why eslint is barking instead of disabling anything.  Do not disable anything.  If it still fails, send me the code that has the failure.